implementation/core/src/main/java/io/smallrye/faulttolerance/core/async/AsyncLogger.java

```diff
@@ -0,0 +1,27 @@
+package io.smallrye.faulttolerance.core.async;
+
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "SRFTL", length = 5)
+public interface AsyncLogger {
+    AsyncLogger log = Logger.getMessageLogger(AsyncLogger.class, AsyncLogger.class.getPackage().getName());
+
+    @Message(id = 2000, value = "CompletionStageExecution started")
+    @LogMessage(level = Logger.Level.TRACE)
+    void completionStageExecutionStarted();
```

Comment:
This just doesn't work well. Typically TRACE/DEBUG are not internationalized as it was not a requirement in the i18n effort. There is little harm in doing it, however here it references class names so the usefullness of localizing this is questionable.

Reply:
Just to explain: the harm is the load this puts on the i18n teams.

Reply:
Ah, good point. I got a bit overzealous here :-) And I probably didn't even know. Thanks, I'll move trace logging to `log.trace("...")`.

Reply:
Hmm, just found that e.g. SmallRye Config or Weld do the same thing -- DEBUG or TRACE logging via i18n interfaces. It's kinda natural to use it all the way through.

Given that the log level is defined statically, I think DEBUG and TRACE messages shouldn't even get to the l10n teams for translation, so the harm should really be none.

Just not sure what to do :-)

Reply:
WildFly also seems to use i18n interfaces for DEBUG and TRACE logging, but that seems to be more of an accident, and vast majority of DEBUG and TRACE logging is directly in the code. So I'll do that too.

Reply:
Done.